images/archive/exporter.go

```diff
@@ -320,10 +320,9 @@ func blobRecord(cs content.Provider, desc ocispec.Descriptor, opts *blobRecordOp
 	if opts != nil && opts.blobFilter != nil && !opts.blobFilter(desc) {
 		return tarRecord{}
 	}
-	path := path.Join("blobs", desc.Digest.Algorithm().String(), desc.Digest.Encoded())
```

Comment:
This was only used below, and `path` was colliding with the `path` import, so I inlined it